pages/index.vue

```diff
@@ -53,14 +38,19 @@ export default {
     }
   },
   methods: {
+    changenews () {
```

Comment:
nice method! 

Why do you subtract 1 why don't we let `newsStart` become the same number as `newsLimit`? Is this an index starting from 0?

Reply:
yes, the index starts from zero, that' why we need -1 for the total number (newsLimit)